pkg/kubelet/winstats/version.go

```diff
@@ -33,33 +33,33 @@ type OSInfo struct {
 func GetOSInfo() (*OSInfo, error) {
 	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Windows NT\CurrentVersion`, registry.QUERY_VALUE)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("open Windows CurrentVersion failed: %w", err)
```

Comment:
small inconsistency here with later messages:

```suggestion
		return nil, fmt.Errorf("getOSInfo, open Windows CurrentVersion failed: %w", err)
```

Reply:
ok